tests/Core/Config/SniffListTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Tests for the \PHP_CodeSniffer\Config --sniffs and --exclude arguments.
+ *
+ * @author  Dan Wallis <dan@wallis.nz>
+ * @license https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Tests\Core\Config;
+
+use PHPUnit\Framework\TestCase;
+use PHP_CodeSniffer\Tests\ConfigDouble;
+
+/**
+ * Tests for the \PHP_CodeSniffer\Config --sniffs and --exclude arguments.
+ *
+ * @covers \PHP_CodeSniffer\Config::processLongArgument
+ */
+final class SniffListTest extends TestCase
+{
+
+
+    /**
+     * Ensure that the expected error message is returned for invalid arguments
+     *
+     * @param string $argument sniffs or exclude
+     * @param string $value    list of sniffs to include / exclude
+     * @param string $message  expected error message text
```

Comment:
@jrfnl consider adding `Squiz.Commenting.FunctionComment.ParamCommentNotCapital` to `phpcs.xml.dist` if this is important.

Reply:
Sounds like a good idea. Let's address that as part of #155. (and figure out why this wasn't done previously/whether that would cause conflicts with the PEAR sniff for the same)